### PR TITLE
finish cleaning up preliminary review

### LINKED ISSUE #3088: Clean up all references to preliminary review
This is part of #3083.

After solving all other dependencies of #3083, all code relating to preliminary review should be unused, so it should also be removed. The cleanup work related to signing can be found in mozilla/addons#32, so that should probably be reopened and resolved.

### PR DESCRIPTION
**Pull request by eviljeff**
Fixes #3088 - the rest of the apps.
There are two commits, the first is everything remaining except `addons`, `constants` and the scripts in `services`.  The second is the big one,

### CODE REVIEW

**Comment by eviljeff** [comment]
Changing `STATUS_UNREVIEWED` to `STATUS_AWAITING_REVIEW` was in part to ensure it was being used correctly - `STATUS_UNREVIEWED` was overloaded to mean both 'awaiting preliminary review' for addons and a file awaiting a review of either type.

File: src/olympia/devhub/tests/test_views.py [hunk 1]
```diff
@@ -847,14 +847,14 @@ def test_no_editor_promo(self):
         assert self.get_pq()('#devhub-sidebar #editor-promo').length == 0
 
     def test_my_addons(self):
-        statuses = [(amo.STATUS_NOMINATED, amo.STATUS_UNREVIEWED),
-                    (amo.STATUS_PUBLIC, amo.STATUS_UNREVIEWED)]
+        statuses = [(amo.STATUS_NOMINATED, amo.STATUS_AWAITING_REVIEW),
+                    (amo.STATUS_PUBLIC, amo.STATUS_AWAITING_REVIEW)]
 
-        for addon_status in statuses:
+        for (addon_status, file_status) in statuses:
```
Comment (EnTeQuAk) [inline src/olympia/devhub/tests/test_views.py]: You don't have to wrap that in parens.

File: src/olympia/versions/tests.py [hunk 2]
```diff
@@ -561,37 +561,12 @@ def test_source_upload_path_utf8_chars(self):
...
-    (amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_UNREVIEWED, True),
-    (amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_LITE, True),
-    (amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_LITE_AND_NOMINATED, True),
-    (amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_NOMINATED, True),
-    (amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_PUBLIC, False),
-    (amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_DISABLED, False),
-    (amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_BETA, False),
-    (amo.STATUS_NOMINATED, amo.STATUS_UNREVIEWED, True),
-    (amo.STATUS_NOMINATED, amo.STATUS_LITE, True),
-    (amo.STATUS_NOMINATED, amo.STATUS_LITE_AND_NOMINATED, True),
```
Comment (EnTeQuAk) [inline src/olympia/versions/tests.py]: omg, that looks glorious. All these possible states, gone :yellow_heart:

File: src/olympia/conf/stage/settings.py [hunk 3]
```diff
@@ -186,7 +186,6 @@
 
 # Signing
 SIGNING_SERVER = env('SIGNING_SERVER')
-PRELIMINARY_SIGNING_SERVER = env('PRELIMINARY_SIGNING_SERVER')
```
Comment (eviljeff) [inline src/olympia/conf/stage/settings.py]: we can!

File: src/olympia/constants/base.py [hunk 4]
```diff
@@ -432,46 +414,46 @@
...
 REVIEWED_OVERDUE_LIMIT = 7
 REVIEWED_PERSONA = 40
 # TODO: Leaving room for persona points based on queue.
 REVIEWED_SEARCH_FULL = 50
-REVIEWED_SEARCH_PRELIM = 51
+_REVIEWED_SEARCH_PRELIM = 51
 REVIEWED_SEARCH_UPDATE = 52
 REVIEWED_THEME_FULL = 60
-REVIEWED_THEME_PRELIM = 61
+_REVIEWED_THEME_PRELIM = 61
```
Comment (eviljeff) [inline src/olympia/constants/base.py]: I'll add a comment about them being deprecated

Comment (eviljeff) [inline src/olympia/constants/base.py]: Added in https://github.com/mozilla/addons-server/pull/3643/commits/94b263f4c0edf9bac6a582d61662ddc41dcefabb

**Pull request merged**